tests/test_api_shim.py

```diff
@@ -211,22 +211,32 @@ def assert_get_spot_value_called(fun, field):
             """
             with patch(spot_value_meth) as gsv:
                 fun()
-                gsv.assert_called_with(
-                    self.asset1,
-                    field,
-                    test_end_minute,
-                    'minute'
-                )
+                if is_new:
+                    gsv.assert_called_with(
+                        asset=self.asset1,
+                        dt=test_end_minute,
+                        field=field,
+                        data_frequency='minute'
+                    )
+                else:
+                    gsv.assert_called_with(
+                        self.asset1,
+                        field,
+                        test_end_minute,
+                        'minute'
+                    )
         # Ensure that data.current(sid(n), field) has the same behaviour as
         # data[sid(n)].field.
         for field in ohlcvp_fields:
             assert_get_spot_value_called(
                 lambda: getattr(bar_data[self.asset1], field),
                 field,
+                False
```

Comment:
To avoid code repetition in BarData.current, I used a partial to store the function (data_portal.get_spot_value vs. data_portal.get_adjusted_value) that we want to use. These are called with keywords, so this assert method must be aware (through the is_new flag) that the new api calls it this way, while the old api just calls with positional args